tools/packager/packager.py

```diff
@@ -32,12 +32,15 @@
 import argparse
 import json
 import utils
+import shlex
 from afu import AFU
 from gbs import GBS
 from metadata import metadata
+from subprocess import Popen, PIPE
```

Comment:
Should probably be `import subprocess` since `check_output` is being called with the `subprocess.` prefix on line 80